src/webui/www/private/scripts/client.js

```diff
@@ -633,9 +633,12 @@ window.addEvent('load', function() {
                             const torrents = response['trackers'][tracker];
                             const host = getTrackerHost(tracker);
                             const hash = genHash(host);
+                            const mergedTorrents = trackerList.has(hash)
+                                ? trackerList.get(hash)['torrents'].concat(torrents)
```

Comment:
There is another problem, the number always increase and never decrease.

Reply:
Yep, will do in the "delete" section.

Reply:
I am thinking about changing the API to make it easier, by grouping tracker by hosts, instead of full url.

Reply:
https://user-images.githubusercontent.com/822628/132765827-20a649fe-a7b7-4f8b-b0b2-de20804fdb21.mp4

Reply:
>I am thinking about changing the API to make it easier, by grouping tracker by hosts, instead of full url.

I'm not sure it is a good idea as it might break existing webAPIusers. So unless necessary, I would prefer not changing the webAPI.

Reply:
It might only break some 3rdparty UI, however, they are accessing an undocumented API anyway.
The alternative is I can rename the field as "tracker_host", and "tracker_host_removed", but I don't think that's necessary anyway.

Reply:
Also, usually WebUI won't use this field directly, since and they are likely to benefit from this API change.

Reply:
>It might only break some 3rdparty UI, however, they are accessing an undocumented API anyway.

IMO undocumented doesn't mean it isn't stable, it could be just someone forgot to document it into WebAPI wiki.

Reply:
>So unless necessary, I would prefer not changing the webAPI.

It doesn't seem easy to fix the issue without breaking the current webapi... I suppose breaking it is anyway.

Reply:
Correct.